learning_journal_basic/learning_journal_basic/tests.py

```diff
@@ -1,29 +1,209 @@
-import unittest
-
+import pytest
+import transaction
+import faker
+import random
 from pyramid import testing
+from .models import (
+    MyEntry,
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+    )
+from .models.meta import Base
+from datetime import datetime
+from pyramid.httpexceptions import HTTPNotFound
+
+TITLE = ["learning journal 1", "learning journal 2", "learning journal 3"]
+
+FAKE = faker.Faker()
 
+ENTRIES = [MyEntry(
+    title=random.choice(TITLE),
+    blog_entry=FAKE.text(100),
+    creation_date=datetime.utcnow()
+    )
+        for i in range(3)]
 
-class ViewTests(unittest.TestCase):
-    def setUp(self):
-        self.config = testing.setUp()
 
-    def tearDown(self):
+@pytest.fixture(scope="function")
+def sqlengine(request):
+    config = testing.setUp(settings={
+        "sqlalchemy.url": "sqlite:///:memory:"
+    })
+    config.include(".models")
+    config.include(".routes")
+    settings = config.get_settings()
+    engine = get_engine(settings)
+    Base.metadata.create_all(engine)
+
+    def teardown():
         testing.tearDown()
+        transaction.abort()
+        Base.metadata.drop_all(engine)
+
+    request.addfinalizer(teardown)
+    return engine
+
+@pytest.fixture(scope="function")
+def new_session(sqlengine, request):
+    session_factory = get_session_factory(sqlengine)
+    session = get_tm_session(session_factory, transaction.manager)
+
+    def teardown():
+        transaction.abort()
+    request.addfinalizer(teardown)
+    return session
+
+def test_model_get_added(new_session):
+    assert len(new_session.query(MyEntry).all()) == 0
+
+@pytest.fixture()
```

Comment:
You should either move fixtures into the section they are used with or create section titles for fixtures so they stand apart from the previous section's tests more.